### PR TITLE
Add comments in seed script.

### PR DESCRIPTION
**Pull request by RishabhJain2018**

### CODE REVIEW

File: scripts/seed.py [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 
 def create_user(is_admin, username=""):
+    """
+    Function to create superuser, participant user and host user and it returns the created user.
```
Comment (TroJan) [inline scripts/seed.py]: Just a suggestion, what if we write the doc string as 
```
Creates superuser, participant use, host user and returns it
```

Same suggestion goes for all the docstrings below.

Comment (RishabhJain2018) [inline scripts/seed.py]: @TroJan sure. I will update accordingly. :-)

**Comment by RishabhJain2018** [comment]
@deshraj @TroJan Please review the PR.

File: scripts/seed.py [hunk 2]
```diff
@@ -54,6 +67,9 @@ def create_challenge_host_team(user):
 
 
 def create_challenges(number_of_challenges=3, host_team=None):
+    """
+    Creates past challenge, on-going challenge, upcoming challenge.
```
Comment (deshraj) [inline scripts/seed.py]: Replace with `Creates past challenge, on-going challenge and upcoming challenge.`

File: scripts/seed.py [hunk 3]
```diff
@@ -98,6 +121,13 @@ def create_challenge(title, start_date, end_date, host_team):
 
 
 def create_challenge_phases(challenge, number_of_phases=1):
+    """
+    Creates a challenge phase and returns it.
+
+    Now create challenge phases for the above created challenge with parameters name,
```
Comment (deshraj) [inline scripts/seed.py]: Replace with `Create challenge phases for the created challenges with parameters name, description, leaderboard_public, is_public, start_date, end_date, challenge, test_annotation, codename.`

File: scripts/seed.py [hunk 4]
```diff
@@ -133,6 +166,9 @@ def create_leaderboard():
 
 
 def create_dataset_splits(number_of_splits):
+    """
+    Creates dataset splits and returns it..
```
Comment (deshraj) [inline scripts/seed.py]: Remove the extra `.`

**Comment by RishabhJain2018** [comment]
@deshraj @TroJan Please review the PR.

File: scripts/seed.py [hunk 5]
```diff
@@ -76,6 +92,13 @@ def create_challenges(number_of_challenges=3, host_team=None):
 
 
 def create_challenge(title, start_date, end_date, host_team):
+    """
+    Creates a challenge.
+
+    Add a challenge with the parameters title, short_description, description, terms_ and_condition,
+    submission_guidelines, evaluation_details, evaluation_script, creator, published, enable_forum,
+    anonymous leaderboard, start_date and end_ date.
```
Comment (deshraj) [inline scripts/seed.py]: `end_ date` to `end_date`

File: scripts/seed.py [hunk 6]
```diff
@@ -76,6 +92,13 @@ def create_challenges(number_of_challenges=3, host_team=None):
 
 
 def create_challenge(title, start_date, end_date, host_team):
+    """
+    Creates a challenge.
+
+    Add a challenge with the parameters title, short_description, description, terms_ and_condition,
```
Comment (deshraj) [inline scripts/seed.py]: `terms_ and_condition` to `terms_and_condition`

File: scripts/seed.py [hunk 7]
```diff
@@ -76,6 +92,13 @@ def create_challenges(number_of_challenges=3, host_team=None):
 
 
 def create_challenge(title, start_date, end_date, host_team):
+    """
+    Creates a challenge.
+
+    Add a challenge with the parameters title, short_description, description, terms_and_condition,
```
Comment (TroJan) [inline scripts/seed.py]: This is redundant. Code is self explanatory. The only need to add comment was just to give an overview of what a function is doing. I think this should not be there @deshraj

File: scripts/seed.py [hunk 8]
```diff
@@ -24,6 +24,12 @@
 
 
 def create_user(is_admin, username=""):
+    """
+    Creates superuser, participant user, host user and returns it.
+
+    To create a user, we must enter username, email and password of the user.
```
Comment (TroJan) [inline scripts/seed.py]: The doc string must tell what a function does in as less words as possible. If we start writing comments that describe what the code is doing, then that is not neat. What say @deshraj

File: scripts/seed.py (hunk 7 above)
Comment (deshraj) [inline scripts/seed.py]: Sounds good.

**Comment by RishabhJain2018** [comment]
@deshraj @TroJan Please review the PR.

**Pull request merged**